ghost/core/core/server/data/schema/schema.js

```diff
@@ -600,6 +600,45 @@ module.exports = {
         updated_at: {type: 'dateTime', nullable: true},
         updated_by: {type: 'string', maxlength: 24, nullable: true}
     },
+    subscriptions: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        type: {
+            type: 'string', maxlength: 50, nullable: false, validations: {
+                isIn: [['free', 'comped', 'paid']]
+            }
+        },
+        status: {
+            type: 'string', maxlength: 50, nullable: false, validations: {
+                isIn: [['active', 'expired', 'cancelled']]
+            }
+        },
+        member_id: {type: 'string', maxlength: 24, nullable: false, unique: false, references: 'members.id', cascadeDelete: true},
+        tier_id: {type: 'string', maxlength: 24, nullable: true, unique: false, references: 'products.id'},
+
+        // These are null if type !== 'paid'
+        cadence: {
+            type: 'string', maxlength: 50, nullable: true, validations: {
+                isIn: [['month', 'year']]
+            }
+        },
+        currency: {type: 'string', maxLength: 3, nullable: true},
+        amount: {type: 'integer', nullable: true},
+        
+        // e.g. 'stripe'
+        payment_provider: {type: 'string', maxlength: 50, nullable: true},
+        // e.g. Stripe Subscription Link
+        payment_subscription_url: {type: 'string', maxlength: 2000, nullable: true},
+        // e.g. Stripe Customer Link
+        payment_user_url: {type: 'string', maxlength: 2000, nullable: true},
+      
+        offer_id: {type: 'string', maxlength: 24, nullable: true, unique: false, references: 'offers.id'},
+      
+        expires_at: {type: 'dateTime', nullable: true},
+        created_at: {type: 'dateTime', nullable: false},
+        created_by: {type: 'string', maxlength: 24, nullable: false},
```

Comment:
@daniellockyer do we keep using the `x_by` fields or do we retire them in favor of using `actions` table? I'm thinking the latter, as there's no good reason to have redundant data like this.

Reply:
They can definitely go, nuke 'em 💥